tensorflow/core/kernels/dml_gather_op.cc

```diff
@@ -319,10 +306,6 @@ class DmlGatherKernel : public DmlKernel {
 
```

Comment:
Is the comment above now stale?

// Because we can't guarantee that
// our output tensor's memory is zero'd, we need to do so manually prior to
// running running gather.

 (for some reason, I can't properly anchor the comment to the actual line above?? GitHub... 🤦‍♀️)